### ISSUE TITLE
Local npm registries / package-lock.json

### ISSUE THREAD

**Issue by dnlkoch** [terrestris/react-geo#455]
Due to local overrides of the npm registry (e.g. to use an internal caching registry like [verdaccio](https://github.com/verdaccio/verdaccio)) made on workspaces that have been used to publish a `react-geo` release, there are at least two different registries listed in the `package-lock.json` in the main repo. This leads to merge conflicts in this file very often, e.g:

```
"resolved": "https://registry.npmjs.org/ansi-styles/-/ansi-styles-3.2.1.tgz"
```
vs.

```
"resolved": "http://10.133.7.204:4873/ansi-styles/-/ansi-styles-3.2.0.tgz"
```

To handle this, I suggest that we either update the internal `publish` script to set the correct npm registry before a release is being published or to disable the generation of the `package-lock.json` completely.

**Comment by KaiVolland** [comment]
IMHO updating the publish script would be much better.

**Comment by weskamm** [comment]
+1 for update of publish script

**Comment by dnlkoch** [comment]
As discussed on the Codesprint, we will remove/ignore the package-lock.json, see [here](https://codeburst.io/disabling-package-lock-json-6be662f5b97d).

**Comment by dnlkoch** [comment]
Fixed via #527.
